app/public/js/c2sScripts.js

```diff
@@ -0,0 +1,9 @@
+(function (global) {
+  'use strict';
+  $('.viewToggle a').not('.checked').attr('href', function() {
+    return this.href + '&open=true';
+  });
+  $('.viewToggle a.checked').attr('href', function() {
+    return this.href = window.location.href.replace(/&open=[^&;]*/,'');
```

Comment:
Use a local copy of `window` as in https://github.com/nhsuk/connecting-to-services/blob/master/app/public/js/geoLocation.js#L9